src/_guides/language/language-tour.md

```diff
@@ -3484,11 +3352,15 @@ void main() {
   // It's also OK to use no <> at all.
   var foo = new Foo();
 
-  // Specifying any non-SomeBaseClass type results in a warning and, in
-  // checked mode, a runtime error.
+  // Specifying any non-SomeBaseClass type results in an error.
   // var objectFoo = new Foo<Object>();
 {% endprettify %}
 
+{% comment %}
+TODO: Confirm that non-SomeBaseClass is an error.
+Perhaps be more specific.
+{% endcomment %}
```

Comment:
I confirm that it is an error: `'Object' doesn't extend 'SomeBaseClass'`. I'd suggest pulling `var objectFoo = new Foo<Object>();` out into it's own excerpt, uncommented, so that we can have the analyzer validate our claim that it is an error.

Would you like for me to make that change in this, or maybe a followup, PR?

If you'd like we can have the analyzer error automatically inlined in the prose (as we've done elsewhere in the docs), but I don't think that is necessary.

Reply:
Yes, if you could write that test, I'd appreciate it. My test fu isn't great. (I agree we don't need to inline the error.)

Reply:
Working on the test now.